src/Symfony/Component/Yaml/Tests/InlineTest.php

```diff
@@ -432,4 +432,13 @@ public function testTheEmptyStringIsAValidMappingKey()
     {
         $this->assertSame(array('' => 'foo'), Inline::parse('{ "": foo }'));
     }
+
+    /**
+     * @expectedException \Symfony\Component\Yaml\Exception\ParseException
+     * @expectedExceptionMessage Unexpected end of line, expected one of ",}".
```

Comment:
when merging on 3.4:
`@expectedExceptionMessage Unexpected end of line, expected one of ",}" at line 1 (near "{abc: 'def'").`